compute/instances/how-to/create-an-instance.mdx

```diff
@@ -48,14 +48,23 @@ This page shows how to create your first Scaleway Instance. An [Instance](/compu
     - Choose an **Availability Zone**, which is the geographical region where your Instance will be deployed.
     - Choose an **Image** to run on your Instance. This can be an operating system, an InstantApp or a custom image. A complete list of all available Linux distributions and InstantApps can be found [here](https://www.scaleway.com/en/imagehub/).
 
-  <Message type="note">
+    <Message type="note">
 
-    InstantApps are not available on [Enterprise Instances](/compute/instances/concepts#enterprise-instance)
+      InstantApps are not available on [Enterprise Instances](/compute/instances/concepts#enterprise-instance)
 
-  </Message>
+    </Message>
 
     - Choose an **Instance type**. Different Instance types have different prices, processing power, memory, SSD disks and bandwidth. 
-    - Add **Volumes**. This is an optional step. Volumes are storage spaces used by your Instances. You can leave the default settings of a minimum local storage, or choose to add more [block](/compute/instances/concepts#block-volumes) or [local](/compute/instances/concepts#local-volumes) storage to your Instance. 
+    - Add **Volumes**. This is an optional step. Volumes are storage spaces used by your Instances. You can leave the default settings of a maximum local storage, or choose how much [local](/compute/instances/concepts#local-volumes) and/or [block](/compute/instances/concepts#block-volumes) storage you want. Your **system volume** is the volume on which your Instance will boot. The system volume can be either a local or a block volume. 
```

Comment:
is it right to change minimum to maximum? Can't the user increase the amount of local storage from the default amount?